blogposts/2020/3.19/lsif-go.md

```diff
@@ -0,0 +1,52 @@
+---
+title: "lsif-go improvements"
+author: Eric Fritz
+publishDate: 2020-08-17T08:00-07:00
+tags: [blog]
+slug: todo
+heroImage: /blog/3.19-todo.png
+published: true
+---
+
+The release of Sourcegraph 3.19 happens to coincide with the [one-point-oh release](https://github.com/sourcegraph/lsif-go/releases/tag/v1.0.0) of [lsif-go](https://github.com/sourcegraph/lsif-go). This release boasts impressive speed improvements that brings intelligence to your Go code faster than ever before.
+
+The code intelligence is hard at work trying to bring the _time to intelligence_ as close to zero as possible. We want your code to always have up-to-date precise code intelligence on the newest commits to help aid users browsing unfamiliar code and code reviewers trying to understand new code in pull requests. Indexing speed is a _huge_ part of this, and we've made a great leap here.
+
+As an illustrative example, checkout the visual difference between indexing our [main repository](https://github.com/sourcegraph/sourcegraph) with lsif-go v0.9 and indexing it with lsif-go v1.0.
+
+#### Old and busted (v0.9.0)
+
+![lsif-go v0.9](https://storage.googleapis.com/sourcegraph-assets/blog/3.19/lsif-go-ss-0.9.gif)
```

Comment:
not sure how easy this would be, but could you make these adjacent in one gif? i had some network loading weirdness so that the old and busted finished first

Reply:
LOL oh no. Yeah I can try to do that.

Reply:
Replaced the gifs with asciicasts.